samples/cellular/udp/src/main.c

```diff
@@ -47,7 +47,9 @@ static void socket_transmission_work_fn(struct k_work *work)
 #if defined(CONFIG_UDP_RAI_ONGOING)
 	/* Let the modem know that we expect to keep the network up longer.
 	 */
-	err = setsockopt(client_fd, SOL_SOCKET, SO_RAI_ONGOING, NULL, 0);
+	int rai = RAI_ONGOING;
+
+	err = setsockopt(client_fd, SOL_SOCKET, SO_RAI, &rai, sizeof(rai));
```

Comment:
What about
```suggestion
	err = setsockopt(client_fd, SOL_SOCKET, SO_RAI, &(int){RAI_ONGOING}, sizeof(int));
```
or is it too ugly for use in a sample?

Reply:
Maybe passing a variable is easier to understand in a sample